rhine failing to build with stackage lts

while attempting to build for lts-19.13 (GHC 9.0.2) we are seeing:
```
--  While building package rhine-0.7.1 (scroll up to its section to see the error) using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.4.1.0_ghc-9.0.2 --builddir=.stack-work/dist
/x86_64-linux/Cabal-3.4.1.0 build lib:rhine --ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/rhine-0.7.1.log

Preprocessing library for rhine-0.7.1..
Building library for rhine-0.7.1..
[29 of 42] Compiling FRP.Rhine.ClSF.Except

/var/stackage/work/unpack-dir/unpacked/rhine-0.7.1-29abae5899c62f87e73244c73349d8619951219e5742628d06b8d59a601844cf/src/FRP
/Rhine/ClSF/Except.hs:15:19: error:
    Not in scope: type constructor or class ‘Empty’
   |
15 |   , safe, safely, Empty, exceptS, runMSFExcept, currentInput
   |                   ^^^^^
```

--- a/rhine/src/FRP/Rhine/ClSF/Except.hs
+++ b/rhine/src/FRP/Rhine/ClSF/Except.hs
@@ -12,7 +12,7 @@ and reexports everything needed from there.
 module FRP.Rhine.ClSF.Except
   ( module FRP.Rhine.ClSF.Except
   , module X
-  , safe, safely, Empty, exceptS, runMSFExcept, currentInput
+  , safe, safely, exceptS, runMSFExcept, currentInput
   )
   where
 

